Cargo.toml

```diff
@@ -6,9 +6,9 @@ edition = "2018"
 license = "MIT"
 
 [dependencies]
-tui = "0.3.0"
-termion = "*"
-
-
+clap = "3.0.0-beta.2"
```

Comment:
Chose beta because of most up to date auto-generated shell completions.

Reply:
I think it's fine, ultimately we produce a compiled binary so the use of beta internally is less of an issue I think.